project/Dependencies.scala

```diff
@@ -10,7 +10,7 @@ object Dependencies {
   val Scala3 = "3.1.3"
   val Scala2Versions = Seq(Scala213, Scala212)
   val ScalaVersions = Dependencies.Scala2Versions :+ Dependencies.Scala3
-  val AkkaVersion = System.getProperty("override.akka.version", "2.8.0")
+  val AkkaVersion = System.getProperty("override.akka.version", "2.8.0+4-94382770-SNAPSHOT")
```

Comment:
need to add the sonatype snapshot repository if we want to merge this

Reply:
Not entirely happy with merging a snapshot dependency to main, so maybe we wait a bit and see if there are more stuff to get in a 2.8.1-M1 of Akka?

Reply:
sure, I agree

Reply:
Bumped in [b9583a0](https://github.com/akka/akka-persistence-r2dbc/pull/377/commits/b9583a0804fb2b7d5e5fe2dc791ff0b6e98bb6d7)